openapi/api.yml

```diff
@@ -87,3 +87,38 @@ paths:
           schema:
             type: string
             format: path
+
+  '/update-compliance':
+    post:
+      tags:
+        - assessment status
+      summary: Update assessment status
+      description: Return an updated assessment status
+      operationId: api.AssessmentsView.status
+      requestBody:
+        description: Assessment status parameters
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: 'components.yml#/components/schemas/UpdateStatus'
+            example:
+              id: '12h6-456j'
+              compliance_status: 'COMPLIANT'
+      responses:
+        200:
+          description: SUCCESS - Assessment status returned
+          content:
+              application/json:
+                schema:
+                  $ref: 'components.yml#/components/schemas/Assessment'
+        404:
+          description: ERROR - Assessment not found
+          content:
+            application/json:
+              schema:
+                $ref: 'components.yml#/components/schemas/GeneralError'
+              example:
+                code: 404
```

Comment:
Have we tested this actually returns a 404? I think it might return a 500 currently if we're throwing an exception? Perhaps we could add a test for it?

Reply:
I think it should return a 404, as 401 is related to authorisation, can we make it return a 404 if assessment_id is not found?

Reply:
Yep, updated! 👍